lib/fuschia/entities/nucleo.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Fuschia.PesquisadorTables.Nucleo do
+  @moduledoc """
+  Esquema que representa um nucleo de Pesquisa no BD
+  """
+  use Ecto.Schema
```

Comment:
```suggestion
  use Fuschia.Schema
```

Temos um `Schema` customizado que insere as informações corfetas de timestamp e futuramente outras configs

Ele pode ser achado em `/lib/fuschia/schema.ex`